### PR TITLE
File Added : Assigment 7 And 8

### PR DESCRIPTION
**Pull request by yogesh141293**

### CODE REVIEW

File: Yogesh_Ghodmare/HTML/Assig.8/Layout.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        <style>
+            body {
+                background-color: rgb(6, 155, 160);
+            }
+            .p1 {
+                text-align: justify;
+            }
+            .divmain {
+                margin-top: 30px;
+
```
Comment (sourabh513) [inline Yogesh_Ghodmare/HTML/Assig.8/Layout.html]: remove extra line

File: Yogesh_Ghodmare/HTML/Assig.8/Layout.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                    consequuntur? Iste totam cumque atque voluptas distinctio laborum, nihil doloremque ad eum, asperiores
+                    omnis, eveniet praesentium quo repellat maxime aliquid ducimus consequuntur id voluptatum possimus
+                    beatae? Dolorem doloribus molestiae earum perspiciatis unde, eveniet ullam illo impedit, aperiam esse,
+                    minima vel! Quae sequi hic repellendus eos voluptatum explicabo. Architecto nostrum aliquam sit delectus
+                    cumque.</p>
+            </div>
+            <div class="div2"></div>
+            <img src="Image5.jpg" alt="index.jpeg">
+        </div>
+        </div>
```
Comment (sourabh513) [inline Yogesh_Ghodmare/HTML/Assig.8/Layout.html]: remove extra (closing div tag)

File: Yogesh_Ghodmare/HTML/Assig7/professionaldetails.htm [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                        <label for="jobtitle">Job Title:</label>
+                        <input type="text" /><br /><br />
+                        <label for="locations">Location:</label>
+                        <input type="text" />
+                        <br /><br />
+                        <label for="Salary">Expected Salary:</label>
+                        <input type="text" />
+                        <br /><br />
+                        <label for="relocate">Relocate?:</label>
+                        <input type="radio" name="relocate" value="yes" /> Yes <input type="radio" name="relocate" value="no" /> No
```
Comment (sourabh513) [inline Yogesh_Ghodmare/HTML/Assig7/professionaldetails.htm]: seprate the line

File: Yogesh_Ghodmare/HTML/Assig7/Userlogin.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                        <label><input type="text" /></label>
+                        <br /><br />
+                        <label for="lname">Last Name:</label>
+                        <label><input type="text" /></label>
+                        <br /><br />
+                        <label for="Password">Password:</label>
+                        <label><input type="password" /></label>
+                        <br /><br />
+                        <label for="gender">Gender:</label>
+                        <input type="radio" name="gender" value="male" /> Male <input type="radio" name="gender" value="female" /> Female
```
Comment (sourabh513) [inline Yogesh_Ghodmare/HTML/Assig7/Userlogin.html]: seprate the lines

File: Yogesh_Ghodmare/HTML/Assig7/userdetails.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                        <input type="text" />
+                        <br /><br />
+                        <label for="lname">Address:</label>
+                        <input type="text" />
+                        <br /><br />
+                        <label for="Password">Password:</label>
+                        <input type="password" />
+                        <br /><br />
+                        <label for="gender">Gender:</label>
+                        <input type="radio" name="gender" value="male" /> Male <input type="radio" name="gender" value="female" /> Female
```
Comment (sourabh513) [inline Yogesh_Ghodmare/HTML/Assig7/userdetails.html:70]: seprate the lines

**Review by SamikshaWani — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**